runtime/src/iree/hal/drivers/hip/registration/driver_module.c

```diff
@@ -35,6 +35,9 @@ IREE_FLAG(
     bool, hip_async_allocations, true,
     "Enables HIP asynchronous stream-ordered allocations when supported.");
 
+IREE_FLAG(bool, hip_disable_async_caching, false,
```

Comment:
bool flags are best as positive - `hip_async_caching`

Reply:
(double negatives aren't good - true to disable and false to enable is always confusing :)